python/dgl/sparse/sparse_matrix.py

```diff
@@ -479,6 +479,62 @@ def is_diag(self):
         """Returns whether the sparse matrix is a diagonal matrix."""
         return self.c_sparse_matrix.is_diag()
 
+    def select(self, dim: int, ids: torch.Tensor or slice):
+        """Returns a sub-matrix selected by a list of ids or by range.
+
+        Parameters
+        ----------
+        dim : int
+            The dim to select from matrix, should be 0 or 1.
+        ids : tensor.Tensor or slice
+            Tensor indicates the row ids to select from matrix.
+            NOTE: The ids can include duplicate values.
+            Slice indicates sub-matrix selected by range [start, end).
+
+        The function has not supported autograd.
+
+        Returns
+        -------
+        SparseMatrix
+            The sub-matrix only contains rows or columns selected.
+
+        Examples
```

Comment:
nit: add a blank line between examples.

Reply:
Fixed, and added the case comments as other examples.